content/articles/understanding-facial-recognition-using-local-binary-pattern-histogram-(lbph)-algorithm/index.md

```diff
@@ -0,0 +1,92 @@
+
+### An Introduction to Facial Recognition
+Facial recognition is the ability to identify faces in an image, and then link them to a particular person. The LBPH is a recognition algorithm, which can recognize human faces. 
+
+Face recognition can be achieved with the help of a learning concept of training and then testing the model with a given set of images. 
+
+Training rules are used to ensure the output decisions criteria and training algorithm can be used to get some input from the data to match the appropriate output type. So, the algorithm and rules are used to simplify the process of learning. The system uses the information gathered from the data to get results. 
+
+The precision and accuracy of the algorithm are verified by using a test set of images. 
+
+
+### Working of the LBPH algorithm
+The LBPH algorithm typically makes use of 4 parameters: 
+- ***Radius:*** The distance of the circular local binary pattern from the center pixel to its circumference and usually takes a value of 1. 
+- ***Neighbors:*** The number of data points within a circular local binary pattern. Usually, the value of 8.
+- ***Grid X:*** The number of cells in the horizontal plane, is usually a value of 8.
+- ***Grid Y:*** The number of cells in the vertical plane, is usually a value of 8.
+
+Given the above-mentioned parameters, LBPH works as follows;
+
+A data set is created by taking images with a camera or taking images that are saved, and then provisioning a unique identifier or name of the person in the image and then adding the images to a database. It is recommended to take many samples from a single individual. A portion of the data set is used for the training of the algorithm, while the rest is used for testing. 
+
+Using a circular neighborhood concept, which takes non-integer pixel points around the selected area. The number of appearances of LBP codes in the image is put together to form a histogram. The classification is then carried out through the calculation of the basic similarities of the histograms under comparison. 
+
+This histogram contains a description of an individual at three different levels: at a pixel-level, labels are combined in a small area to create a regional level, the regional histograms in combination build a general description of the person. 
+
+### Two modes of operation of face recognition 
+The face recognition algorithm generally operates in one of two modes: 
+
+1.	*Authentication of a facial image:* This mode does facial recognition by a 1x1 comparison. The comparison is done between an input image and a specific image within the database. In many cases, this is the face that requires authentication at the time of this mode of facial recognition.
+
+2.	*Face recognition:* in this mode, it is a 1xN, a comparison of the input face image with all the pictures that have been saved in the database to output the images of the user which conforms to the input face image.
+
+### The stages of face recognition using LBPH 
```

Comment:
More value would be added in this section by using images/illustrations. Consider finding a couple of them to help readers visualize sections like the LBP operation and how histograms are extracted.